pkg/apis/kubeone/types.go

```diff
@@ -255,7 +255,36 @@ type ClusterNetworkConfig struct {
 	// CNI
 	// default value is {canal: {mtu: 1450}}
 	CNI *CNI `json:"cni,omitempty"`
-}
+	// KubeProxy
+	KubeProxy *KubeProxyConfig `json:"kubeProxy,omitempty"`
+}
+
+type KubeProxyConfig struct {
+	IPVS     *IPVSConfig `json:"ipvs"`
+	IPTables *IPTables   `json:"iptables"`
+}
+
+type IPVSConfig struct {
+	// ipvs scheduler
+	Scheduler string `json:"scheduler"`
```

Comment:
Should we add some validation for this field?

Reply:
I'm not sure, original KubeProxyConfiguration doesn't have it. And default empty would work as well.